client/src/store/new-request-module.ts

```diff
@@ -1344,7 +1351,7 @@ export class NewRequestModule extends VuexModule {
   }
 
   get requestTextFromValue () {
-    if (this.request_action_cd) {
+    if (this.request_action_cd && this.requestTypeOptions.find(req => req.value === this.request_action_cd)) {
```

Comment:
This line is too long (per linter), and several others in this file.
Please run `npm run lint` to check.

Reply:
oh interesting, how did this pass the gha CI check? I will run it locally and update

Reply:
Maybe because it's a lint warning not an error?